lib/reducers/ui.js

```diff
@@ -38,6 +38,7 @@ const initial = Immutable({
   cursorBlink: false,
   borderColor: '#333',
   fontSize: 12,
+  fontWeight: 500,
```

Comment:
@ppot I fixed that but the problem I am facing is that when I tried adding font-weight property ), it seems like the hterm package doesn't support font-weight property. Should I overwrite hterm's package set method? Any other way around it? Any suggestion would be appreciated.